platformLibs/src/platform/mingw/winscard.def

```diff
@@ -0,0 +1,15 @@
+package = platform.winscard
+depends = windows
```

Comment:
Add: `noStringConversion = SCardListReaders`

This returns a `msz` (multi-string) which is a list of null terminated strings, then terminated by a null.

Reply:
Then same should be for `SCardLocateCards`.
Done.